x-pack/elastic-agent/elastic-agent.reference.yml

```diff
@@ -11,8 +11,9 @@ outputs:
   default:
     type: elasticsearch
     hosts: [127.0.0.1:9200]
-    username: elastic
-    password: changeme
+    api_key: "example-key"
+    # username: elastic
```

Comment:
Nit: leave out the spaces in front of the key, like this it is easier for users to delete the `#` and have a valid config.